mtp_transaction_uploader/settings.py

```diff
@@ -50,8 +50,10 @@
 
 DS_NEW_FILES_DIR = os.environ.get('DS_NEW_FILES_DIR', '/tmp/ds_new_files')
 
-NOMS_AGENCY_ACCOUNT_NUMBER = os.environ.get('NOMS_AGENCY_ACCOUNT_NUMBER', 'PPPPPPPP')
-NOMS_AGENCY_SORT_CODE = os.environ.get('NOMS_AGENCY_SORT_CODE', 'XXXXXX')
+# fallback account is for tests
+NOMS_AGENCY_ACCOUNT_NUMBER = os.environ.get('NOMS_AGENCY_ACCOUNT_NUMBER', '67175315')
+NOMS_AGENCY_SORT_CODE = os.environ.get('NOMS_AGENCY_SORT_CODE', '123456')
+
```

Comment:
I prefer to keep anything tests rely on out of the code base - would this be better using override settings in the tests themselves? 

Although this could be done later, not a blocking change.

Reply:
i agree, but i was trying to keep the changes to a minimum. the tests already relied on PPPPPPPP/XXXXXX being here

Reply:
Cool, I guessed as much.